packages/axelar-wasm-std/Cargo.toml

```diff
@@ -50,17 +50,19 @@ serde_json = "1.0.89"
 serde_with = { version = "3.11.0", features = ["macros"] }
 sha3 = { workspace = true }
 stellar-xdr = { workspace = true }
+starknet-types-core = { workspace = true }
 strum = { workspace = true }
 sui-types = { workspace = true }
 thiserror = { workspace = true }
 valuable = { version = "0.1.0", features = ["derive"] }
+crypto-bigint = { version = "0.5.5", features = ["rand_core"] }
```

Comment:
I don't think we need this dep, there is already a u256 type available

Reply:
Done. I changed the test U256 randomisation and removed this dependency.